bundle/terranova/db/db.go

```diff
@@ -0,0 +1,170 @@
+package db
```

Comment:
This package looks more like state management, shall it be called like this?

Reply:
"state management" for me would involve more things, like uploading/downloading/validating/repairing state. This package is just a simple storage.

Reply:
Then storage could be a better name I guess, `db` (at least for me) suggest actual database which also involves other things too